Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -2055,10 +2059,10 @@
 
 #if ENABLE(LOCKDOWN_MODE_TELEMETRY)
 (with-filter (require-not (require-entitlement "com.apple.security.cs.allow-jit"))
-    (allow syscall-unix (with report) (with telemetry) (syscall-unix-blocked-in-lockdown-mode))
-    (allow syscall-unix (with report) (with telemetry) (syscalls-rarely-used))
+    (deny syscall-unix (with telemetry) (syscall-unix-blocked-in-lockdown-mode))
+    (deny syscall-unix (with telemetry) (syscalls-rarely-used-blocked-in-lockdown-mode))
     (when (equal? (param "CPU") "arm64")
-        (allow syscall-unix (with report) (with telemetry) (syscall-unix-apple-silicon))))
+        (deny syscall-unix (with report) (with telemetry) (syscall-unix-apple-silicon))))
```

Comment:
Should move `(with report)` now that this is blocking.

Reply:
> Should move `(with report)` now that this is blocking.

That is a good catch. Will fix.

Thanks for reviewing!